exporter/kinesisexporter/exporter.go

```diff
@@ -48,22 +48,24 @@ func (e Exporter) Shutdown(context.Context) error {
 }
 
 // ConsumeTraceData receives a span batch and exports it to AWS Kinesis
-func (e Exporter) ConsumeTraceData(c context.Context, td consumerdata.TraceData) error {
-	pBatch, err := jaegertranslator.OCProtoToJaegerProto(td)
+func (e Exporter) ConsumeTraces(_ context.Context, td pdata.Traces) error {
```

Comment:
I think we should not use `_` for the context argument even if it is unused. It introduces a slight chance of someone creating a new context down the function in future instead of using this one. It's a very minor thing but worth it IMO. @bogdandrutu